### DISCUSSION TITLE
Rerun build after edit podfile using VNC remote

### DISCUSSION THREAD

**Discussion by jboutonville** [codemagic-ci-cd/codemagic-docs#2187] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

After connecting to the build machine using VNC and editing the podfile, how do I rerun the build on the build machine?

### Expected results

From Code Magic documentation:
Using the remote build machine
You can use the remote session to reproduce all commands run during the build, rerun the build with a different configuration, or set up platform-specific files, e.g. CocoaPods.

    To access the clone of your project on the VM, run cd $CM_BUILD_DIR in the terminal.

    The command printenv prints all the environment variables exported during the build on Linux/macOS. Use dir env: on Windows to achieve the same in PowerShell. You can see some of Codemagic built-in environment variables explained [here](https://docs.codemagic.io/building/environment-variables/#codemagic-read-only-environment-variables).

    The sudo command is available on Linux/macOS so you can execute all commands with root privileges. PowerShell on Windows machines is already running in privileged mode.

### Actual results

Don't know how to rerun the build on the remote build machine after connecting to it.

**Comment by icarusdust** [comment]
Hi @jboutonville, after editing  and saving the Podfile of your project in virtual machine via VNC, you can run the build command or whatever command you need to run in the terminal as long as the session is not expired (either by opening the terminal in VNC or starting SSH session at the same time in your local machine terminal, then run the build command there).

**Comment by icarusdust** [comment]
I am closing the thread due to inactivity. You can re-open it by replying to it or start a new thread.
